### DISCUSSION TITLE
Are some features language specific?

### DISCUSSION THREAD

**Discussion by Janscode** [deepgram/community#571] [General help] [answered]
I'm curious if any features are language specific. I see from [this page](https://developers.deepgram.com/docs/models-languages-overview) that some models are limited to specific languages, but I can't find any information about features such as Diarization.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @Janscode, Diarization is available for all languages. [This page](https://developers.deepgram.com/docs/stt-pre-recorded-feature-overview) has more info about other features.

**Reply by Janscode** [reply]
Thanks!
